source/commands.rst

```diff
@@ -27,8 +27,9 @@ Gameplay
     /heal,"[-s] [<player>]","Heal yourself or another player. ``-s`` can be optionally specified to have no message emitted."
     /slay,"[-s] [<player>]","Slay yourself or another player. ``-s`` can be optionally specified to have no message emitted."
     /locate,"[<player>]","Have your compass point to another player (if specified), or pointed to spawn if no player is specified."
-    /stack,"","Organize your inventory and automatically stack items."
-    /;,"","An alias for ``/stack``."
+    "
+    | /stack
+    | /;",,"Organize your inventory and automatically stack items."
```

Comment:
I added the aliases into the same row. Some commands had the aliases listed in a seperate row, some didn't have any aliases listed. This makes this table slightly more condensed, but at least now everything is listed. 
![grafik](https://github.com/EngineHub/WorldGuardDocs/assets/21148213/6c7138f3-16ee-49a1-83fc-4a8d16a5dc64)